### PR TITLE
Speed up testsuite

### PR DESCRIPTION
**Pull request by stgraber**

### CODE REVIEW

File: test/suites/basic.sh [hunk 1]
```diff
@@ -478,10 +478,6 @@ test_basic_usage() {
 
   [ "${REBOOTED}" = "true" ]
 
-  # Workaround for LXC bug which causes LXD to double-start containers
-  # on reboot
-  sleep 2
```
Comment (brauner) [inline test/suites/basic.sh]: What the hell was that all about? :) (just curious)

**Comment by brauner** [comment]
Thanks!

File: test/suites/basic.sh (hunk 1 above)
Comment (stgraber) [reply test/suites/basic.sh]: IIRC, that was LXC still restarting the container when a post-stop hook failed

Comment (brauner) [reply test/suites/basic.sh]: Oh right, I think I fixed that a while ago.

**Pull request merged**